mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.tmpl

```diff
@@ -6848,6 +6848,7 @@ resource "google_compute_instance" "foobar" {
   name         = "%s"
   machine_type = "e2-medium"
   zone         = "us-central1-a"
+  allow_stopping_for_update = true
```

Comment:
My conclusion after looking is that these tests should not be changed. allow_stopping_for_update should not be needed on these tests because these tests do not update. If there is no way to check that this will effect only update plans we should avoid making this change.

This would break existing customer terraform deployments where `allow_stopping_for_update` is not specified.